jdk/src/solaris/back/linker_md.c

```diff
@@ -100,14 +108,12 @@ dbgsysBuildLibName(char *holder, int holderlen, const char *pname, const char *f
 {
     const int pnamelen = pname ? strlen(pname) : 0;
 
-    *holder = '\0';
```

Comment:
I had assumed that `dll_build_name` would do the equivalent of this on failure, but as I look at it again, it does not. Could you fix that?

Reply:
Done. I could have put it in the if, but this way we're covered even if strtok_s fails and returns a null, or if anything else unforeseen happens.